configure.ac

```diff
@@ -7936,7 +7936,7 @@ then
     # available on x86_64. However, performance of the HACL SIMD128 implementation
     # isn't great, so it's disabled on ARM64.
     AC_MSG_CHECKING([for HACL* SIMD128 implementation])
-    if test "$UNIVERSAL_ARCHS" == "universal2"; then
+    if [[ "$build_cpu" == "aarch64" && "$build_vendor" == "apple" ]]; then
```

Comment:
Please keep the "UNIVERSAL_ARCHS" test as well and use `test` instead of `[[` as it's more portable.